test/pkg/environment/aws/expectations.go

```diff
@@ -422,13 +425,13 @@ func (env *Environment) ExpectInstanceProfileDeleted(instanceProfileName, roleNa
 		InstanceProfileName: aws.String(instanceProfileName),
 		RoleName:            aws.String(roleName),
 	}
-	_, err := env.IAMAPI.RemoveRoleFromInstanceProfile(removeRoleFromInstanceProfile)
+	_, err := env.IAMAPI.RemoveRoleFromInstanceProfile(context.Background(), removeRoleFromInstanceProfile)
 	Expect(awserrors.IgnoreNotFound(err)).To(BeNil())
 
 	deleteInstanceProfile := &iam.DeleteInstanceProfileInput{
 		InstanceProfileName: aws.String(instanceProfileName),
 	}
-	_, err = env.IAMAPI.DeleteInstanceProfile(deleteInstanceProfile)
+	_, err = env.IAMAPI.DeleteInstanceProfile(context.Background(), deleteInstanceProfile)
```

Comment:
We need to make sure we are using the environment context 

```suggestion
	_, err = env.IAMAPI.DeleteInstanceProfile(env.Context, deleteInstanceProfile)
```